### ISSUE TITLE
Crew Management - Employee Schedule Schedules update

### ISSUE THREAD

**Issue by JasonSchneider19** [Sanjel-Energy-Services/eService_WorkItems#88]
Currently in the Crew Mgmt - Employee Schedule, we have 15&6 and 5&2. 
![image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/b3f4e317-252f-4b84-a922-c628f53653dd)
There are a few more that need to be added:
10/4, 21/14 and 7/3-7/4

For 10/4 - This means that a person is working 10 days, gets 4 days off, works 10, 4 off etc
![image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/4fe35405-a3c7-4caf-894b-484e8745d70d)

21/4 - This means that a person is working 21 days, gets 4 days off, works 21, 4 off etc
![image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/b49d1b6c-c2b4-40f2-9eeb-50784cf811c0)

7/3-7/4 - This means that a person is working 7 days, gets 3 days off, works 7, 4 off and repeat - this one needs additional configuration.
![image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/e650d98b-25f2-456d-8c49-0780e071ee76)
[Rotation Schedule - Logistics.xlsx](https://github.com/Sanjel-Energy-Services/eService_WorkItems/files/12600044/Rotation.Schedule.-.Logistics.xlsx)

**Comment by awangatsanjel** [comment]
Yes. This will a task on my side to create a mechnisam to support 7/3 & 7/4 schedule.

**Comment by arcurve-cheryl-chan** [comment]
OK, thanks Adam! I'll add the Sanjel tag to this and move it to the Ready column.

**Comment by arcurve-cheryl-chan** [comment]
We might need to update the Crew Mgmt - Employee Schedule page to show all the options as there will be too many to just have on the page as currently. @JasonSchneider19  do you have a preference on how you'd like this displayed?

**Comment by arcurve-cheryl-chan** [comment]
Jason will be getting some UX feedback for this today.

**Comment by JasonSchneider19** [comment]
@arcurve-cheryl-chan the preference would be to have a "Shift Groups" that would create a hover to see all the colors.
When hovering, would like it in the following order:


![Image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/235b4251-98a5-43d1-ac22-3c0fcced8bf1)

This is in order of usage and importance.
There are currently duplicates in the list and the colors need to be different.
For 7&3/7&4 groups - please use Navy Ble/Dark Green
Other groups pick some colors with all being different..... 
If we cannot hover, let me know

**Comment by awangatsanjel** [comment]
@JasonSchneider19 You can pick up the colors for each group. I can update the master data.

**Comment by JasonSchneider19** [comment]
@awangatsanjel @arcurve-cheryl-chan 
Here are my color choices:
•	15/6 x 3 – Leave colors as is
•	7&3/7&4 (need 3 colors)
	    Blue - 2C365E (44,54,94)
	    Green - 0B6E4F (11,110,79)
	    Engineering Orange - D00000 (208,0,0)
•	21 & 14 (Need 5 Colors)
	    Mint Green - BEE9E8 (190,233,232)
	    Moonstone - 62B6CB (98,182,203)
	    Indigo Dye - 1B4965 (27,73,101)
	    Columbia Blue - CAE9FF ( 202,233,255)
	    Picton Blue - 5FA8D3 (95,168,211)
•	5&2 (leave black)
•	10&4 (Need 2 colors)
	    Byzantium - 7A306C (122,48,108)
	    Cool Grey - 8E8DBE (142,141,190)

**Comment by JasonSchneider19** [comment]
@arcurve-cheryl-chan 
If we are doing clicks to load the boxes, a click here would work too. Just have a crew legend name, click on it and bring up all the colors and shifts.

**Comment by sairamk22** [comment]
@JasonSchneider19 I made all the changes mentioned. I've added crew and shiftgroups to a accordion. Please review this and let me know if this works

![Image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/111007738/897eea14-56e1-45f3-a6a6-893aa850880a)

**Comment by JasonSchneider19** [comment]
@sairamk22 
I think all we really need here is the legend. Keep this simple for reference

**Comment by JasonSchneider19** [comment]
@sairamk22 
Can we change this back to something like: (this is what i meant by the above comment)

![Image](https://github.com/Sanjel-Energy-Services/eService_WorkItems/assets/95711838/409986c9-94ca-4b88-9176-190bb32eab37)

lets keep this simple please

**Comment by sairamk22** [comment]
@JasonSchneider19  My bad. I will fix it right away

**Comment by cameliabalan** [comment]
Tested in the latest build #112 and the Shift Groups button displays the shifts and groups as shown in the picture above.
